types/react-native/index.d.ts

```diff
@@ -7974,6 +7984,8 @@ export interface UIManagerStatic {
      * has been completed in native. If you need the measurements as soon as
      * possible, consider using the [`onLayout`
      * prop](docs/view.html#onlayout) instead.
+     * 
+     * @deprecated Use `HostComponent.measure` instead.
```

Comment:
Our messages for this have said 

use ref.measure instead.

We try to avoid saying HostComponent when we can. Saying ref is more clear